ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -579,7 +579,7 @@ def _get_columns(
                 unique_columns.extend(constraint.get("column_names"))
 
         table_columns = []
-        columns = inspector.get_columns(table, schema)
+        columns = inspector.get_columns(table, schema, db_name=self.config.database)
```

Comment:
@ayush-shah given we are supporting multiple databases we need to pass this from get_inspectors right?

Reply:
so we have removed get_inspectors -> we will have to get this from _get_databases and create a class scoped variable.
also we don't support self.config.database based on latest schema changes so we will have to change those as well.
we use self.service_connection.database.